CHANGELOG.md

```diff
@@ -5,7 +5,97 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [v1.1.0] - 2018-08-21
+## [1.2.0] - 2018-12-19
+
+This version contains 155 contributions from 11 contributors. There are 113 files changed; 19,085 insertions; and 4,323 deletions.
+
+### Added 
+
+- Cancel queries on Context.Done() ([#1773](https://github.com/pilosa/pilosa/pull/1773))
+- Union In Place ([#1766](https://github.com/pilosa/pilosa/pull/1766), [#1774](https://github.com/pilosa/pilosa/pull/1774))
+- Import benchmarking ([#1771](https://github.com/pilosa/pilosa/pull/1771))
+- Add GroupBy() Filter ([#1753](https://github.com/pilosa/pilosa/pull/1753))
+- Add /internal/translate/keys endpoint ([#1751](https://github.com/pilosa/pilosa/pull/1751))
+- CircleCI: Add race detector to parallel build, default to Go 1.11. ([#1756](https://github.com/pilosa/pilosa/pull/1756))
+- Add distributed tracing. ([#1684](https://github.com/pilosa/pilosa/pull/1684))
+- Add NoStandardView field option ([#1733](https://github.com/pilosa/pilosa/pull/1733))
+- Import roaring endpoint accepts a list of views ([#1738](https://github.com/pilosa/pilosa/pull/1738))
+- Add some stat tracking to roaring implementation ([#1743](https://github.com/pilosa/pilosa/pull/1743))
+- Cluster tests ([#1717](https://github.com/pilosa/pilosa/pull/1717))
+- Allow backslash, carriage return in PQL strings ([#1713](https://github.com/pilosa/pilosa/pull/1713))
+- Add base system, curl and jq for debug and checks ([#1707](https://github.com/pilosa/pilosa/pull/1707))
+- Add "Rows" and "GroupBy" functionality ([#1647](https://github.com/pilosa/pilosa/pull/1647))
```

Comment:
I guess just a note that going forward we should be consistent with double quotes vs. back-ticks (see next line).

Reply:
standardizing on backticks